doc/debugging.rst

```diff
@@ -14,33 +14,91 @@ This allows breakpoints and the execution flow to be viewed and controlled right
 Debugging with ipdb
 ===================
 
-Spyder offers the following debugging features integrated into the native GUI:
+You can fully control debugger execution from the :guilabel:`Debug` menu, :guilabel:`Debug toolbar` and via configurable keyboard shortcuts, along with the standard ``ipdb`` `console commands`_.
```

Comment:
We don't do this for the other panes with shortcuts; they are all listed in the shortcut summary GUI and in preferences, along with on the tooltips of their toolbar buttons and next to their respective commands in the Debug menu. Instead of a bespoke table, if we do want to show something it would be better to just show a screenshot of the Debug menu, since it shows not only the commands and the shortcuts, but also their icons and where they can be found in the interface.

Reply:
Ok, sounds good to me. @juanis2112, what do you think?

Reply:
I don't see this added yet

Reply:
I realised there is a gif right under this sentence. Not sure if the screenshot fits somewhere else. What do you think?

Reply:
Good point, I don't see anywhere it could easily fit. The one thing you could do would be to move the GIF above this paragraph (which would make it at the very top of the section) and the screenshot below, but that's not ideal and would lead to a high screenshot density. My suggestion would just be to not add it, since its not critical (since users can always just click the debug menu themselves to see it, if they need to), but I'm not sure what @ccordoba12 thinks.

Reply:
> My suggestion would just be to not add it

Ok, I'm fine with that, so let's leave it out.